install/cws-ui/deployments.ftl

```diff
@@ -891,7 +897,7 @@
 				" onClick=\'enableDisable(\""+listWorkers[i].id.toString()+"\");\' "+(listWorkers[i].accepting_new ? "checked" :"")+"/>"+
 				"<label for='"+listWorkers[i].id+"-box'"+( (listWorkers[i].status == 'down') ? " class='w-down'" : '')+">"+listWorkers[i].name+ 
 				(listWorkers[i].cws_install_type == 'console_only'? '*' : '') +"</label>"+
-				"<span id='"+listWorkers[i].id+"-msg'>"+
+				"<span id='"+listWorkers[i].id+"-msg'></span>" +			
```

Comment:
@zonagit and comment on how the <span> tag can be utilized for content provided by the adaptation code..

Reply:
@galenatjpl  The need for this came from m20: Add a filter box to the left of the select all checkbox to be able to filter the workers by scaling class (so if we have 100+ workers we can pick just those in asg8 and select a few of them for example)